### PR TITLE
test fixes and stuff

### PR DESCRIPTION
**Pull request by jmccarthy14**
@iproctor

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -21,7 +21,7 @@ var MAX_PARALLEL_TASKS = 5;
  *      ownership of a key, for example if another node now owns it.  Cleanup.
  *      Also takes a 'done' callback.
  */
-function Sevnup(loadVNKeysFromStorage,
+function Sevnup(loadVNKeysFromStorage, // jshint ignore:line
```
Comment (jwolski) [inline index.js]: why are you jshint ignoring?

Comment (jmccarthy14) [inline index.js]: gives me a too many arguments complaint.  the alternative is to just make an amorphous object with an equal amount of arguments that you just have to pack and unpack, so it's moot.

Comment (iproctor) [inline index.js]: but those are named params and so are easier to read

**Comment by iproctor** [comment]
ok, but update readme

**Comment by iproctor** [comment]
just merge it

**Pull request merged**